src/OpenApi/Generator/Normalizer/NormalizerGenerator.php

```diff
@@ -61,27 +61,23 @@ protected function normalizeMethodStatements(Expr\Variable $dataVariable, ClassG
      */
     protected function createSupportsNormalizationMethod(string $modelFqdn, string $proxyFqdn, bool $useProxy)
     {
-        $statements = [];
-
-        if ($useProxy) {
-            $statements[] = new Stmt\Return_(new Expr\BinaryOp\Identical(
-                new Expr\FuncCall(new Name('get_class'), [new Expr\Variable('data')]),
-                new Scalar\String_($proxyFqdn)
-            ));
-        } else {
-            $statements[] = new Stmt\Return_(new Expr\BinaryOp\Identical(
-                new Expr\FuncCall(new Name('get_class'), [new Expr\Variable('data')]),
-                new Scalar\String_($modelFqdn)
-            ));
-        }
+        $fqdn = $useProxy ? $proxyFqdn : $modelFqdn;
```

Comment:
I made a little mistake with that proxy feature in this Generator, could you please make same as in https://github.com/janephp/janephp/blob/master/src/JsonSchema/Generator/Normalizer/NormalizerGenerator.php#L65 with both condition ?

For example: `is_object($object) && (get_class($object) === FQDN_MODEL || get_class($object) === FQDN_PROXY)` (don't worry about parenthesis, `php-parser` will handle them)

Reply:
Yeah sure ! 

How can i fix the tests because there is a lot of files generated ? ^^